pkg/printer/text.go

```diff
@@ -48,15 +50,26 @@ func (c *textPrinter) Print(results []judge.Result) error {
 	ruleSet := ""
 	w := tabwriter.NewWriter(c.commonPrinter.outputFile, 10, 0, 3, ' ', 0)
 
+	labels := ctx.Value(ctxKey.LEBELS_CTX_KEY).(*bool)
```

Comment:
I think it would be a good idea to use a helper function(s) to manage the data in the context WDYT @femrtnz 
```
func GetLabels(ctx context.Context) (*bool, error) {
	if v := ctx.Value(LABELS_CTX_KEY); v != nil {
		return ctx.Value(LABELS_CTX_KEY).(*bool), nil
	}
	return nil, error.New("labels not found in context.")
}
```

Reply:
yeah agree! fizex on last commit